### DISCUSSION TITLE
Pin GitHub PR

### DISCUSSION THREAD

**Discussion by rohitgr7** [community/community#8301] [Pull Requests]
Is it possible to pin GitHub PRs?
We have a bug fix release the PR for it stays for a week on hold. Need to pin it up so that its available at the top.
Can't find a way if it's possible. Any pointers would be helpful.

**Comment by mntlty** [comment]
@rohitgr7 you could add a label to the PR and filter for PRs that include that label, as well as bookmark the URL with the query, for example in a slack channel

**Reply by rohitgr7** [reply]
well yes, this is a workaround...
but still, I'd like to have such a bookmark on GitHub available not just for a specific group (or me) but for all the developers using that repo.
